providers/shared/components/apigateway/id.ftl

```diff
@@ -102,9 +102,15 @@ object.
                 "Default" : []
             },
             {
-                "Names" : "Authentication",
+                "Names" : ["AuthorisationModel", "AuthorizationModel", "Authentication"],
                 "Types" : STRING_TYPE,
-                "Values" : ["IP", "SIG4ORIP", "SIG4ANDIP"],
+                "Values" : [
+                    "IP",
+                    "SIG4ORIP", "SIG4_OR_IP",
```

Comment:
Should SIG4 move to the aws provider as an extension attribute? Its very specific to AWS

Reply:
Probably but it has been like that so long removing it would require a breaking change.

Reply:
I could make the others I'm adding AWS specific and leave the unhyphenated ones as is?

Reply:
I've moved the new `SIG4_*` values to the AWS provider and prefixed them. We can plan to remove the shared values in a future breaking change - https://github.com/hamlet-io/engine/issues/1997

Reply:
You can also add shared:<value> in the aws provider to keep them using the existing value while also adding the prefixed version 

https://github.com/hamlet-io/engine-plugin-aws/blob/bb0c68eb47dd6bc8f275808979d655af1fb2685e/aws/components/ecs/id.ftl#L101 

Has an example of that for the awsvpc network mode on containers